ci-operator/step-registry/upi/conf/vsphere/zones/upi-conf-vsphere-zones-commands.sh

```diff
@@ -105,7 +105,6 @@ fi
 
 ${platform_required} && cat >> "${install_config}" << EOF
 baseDomain: $base_domain
-featureSet: TechPreviewNoUpgrade
```

Comment:
@rvanderp3 Hi, could you help to take a look if we can remove TP featureSet in UPI workflow, thanks.

Reply:
yep! we can.  great catch.

/lgtm
/approve